warmup.krun

```diff
@@ -140,21 +140,27 @@ SKIP = [
 N_EXECUTIONS = 10  # Number of fresh processes.
 
 if sys.platform.startswith("linux"):
+    # Assumes systemd is used for init.
     PRE_EXECUTION_CMDS = [
-        "sudo service cron stop",
-        "sudo service atd stop",
-        "sudo service postfix stop",
-        "sudo service networking stop",
+        # Stop network first. If an interface accepts a DHCP lease during one
+        # of the later commands below, it can cause the command to be
+        # "cancelled" by systemd. Bringing the network itself down can fail in
+        # the same way, so try up to 5 times (with sleeps between) before
+        # finally giving up.
+        "sudo sh -c 'for i in 1 2 3 4 5; do systemctl stop networking && break; sleep 10; false; done'",
```

Comment:
E.g. the lease is renewed, or the initial lease still didn't happen (I think the latter is what happened in the case we saw).

I pretty much did copy the loop, just I didn't want it to do this forever. Well, we could make it run forever, if we really wanted. Do we?